packs/data/equipment.db/hyldarfs-fang.json

```
@@ -79,18 +79,18 @@
                 "selector": "{item|_id}-damage"
             },
             {
-                "key": "Note",
-                "outcome": [
-                    "criticalSuccess"
-                ],
-                "selector": "{item|_id}-damage",
-                "text": "Hyldarf's Fang also deals [[/r 3d10[persistent,fire]]] damage on a critical hit.",
-                "title": "{item|name}"
+                "category": "persistent",
+                "critical": true,
+                "damageType": "fire",
+                "diceNumber": 3,
+                "dieSize": "d10",
+                "key": "DamageDice",
+                "selector": "{item|_id}-damage"
             },
             {
                 "key": "Note",
                 "selector": "{item|_id}-damage",
-                "text": "Fire damage dealt by this weapon (including persistent fire damage and damage from Hyldarf's Venom) ignores the target's fire resistance.",
+                "text": "PF2E.SpecificRule.HyldarfsFang.Note",
```

```suggestion
                "text": "PF2E.WeaponPropertyRune.greaterFlaming.Note.criticalSuccess",
```
#7701 changes the text to drop the persistent damage portion since it's automated now.